iroh/src/disco.rs

```diff
@@ -143,6 +144,8 @@ pub enum SendAddr {
     Udp(SocketAddr),
     /// Relay Url.
     Relay(RelayUrl),
+    /// Node Id
+    NodeId(NodeId)
```

Comment:
I think you'll want this to be sth like `WebRtc(SocketAddr)`. Not sure if there's any other info need to send webrtc data besides the socket addr. Perhaps a channel ID?